lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -197,6 +217,15 @@ object Mocks {
         nodeIdentifier = Some.nodeIdentifier,
         signature = Some.signature
     )
+
+    fun postBlockRequest() = PostBlockRequest(
+        accountNumber = "0cdd4ba04456ca169baca3d66eace869520c62fe84421329086e03d91a68acdb",
```

Comment:
Please use the constants defined in this class to keep this maintainable. 

`Some.accountNumber`